### ISSUE TITLE
Skin Object scaffording not working

### ISSUE THREAD

**Issue by gncube** [UpendoVentures/generator-upendodnn#29]
## Describe the bug
The Upendo Ventures DNN project generator for creating Skin Objects is throwing a javascript error.

events.js:167
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find module 'uuid'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:594:15)
    at Function.Module._load (internal/modules/cjs/loader.js:520:25)
    at Module.require (internal/modules/cjs/loader.js:650:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (C:\Users\****\AppData\Roaming\npm\node_modules\generator-upendodnn\generators\skinobject\index.js:4:14)
    at Module._compile (internal/modules/cjs/loader.js:702:30)    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
Emitted 'error' event at:
    at Immediate.setImmediate (C:\Users\***\AppData\Roaming\npm\node_modules\generator-upendodnn\node_modules\yeoman-generator\lib\index.js:433:22)
    at runCallback (timers.js:696:18)
    at tryOnImmediate (timers.js:667:5)    at processImmediate (timers.js:649:5)

## To Reproduce
1. Run: yo upendodnn
2. Select the Skin Object project
3. Add local URL and path

## Expected behavior
Project is created just like the other projects which are working.
## Errors

events.js:167
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find module 'uuid'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:594:15)
    at Function.Module._load (internal/modules/cjs/loader.js:520:25)
    at Module.require (internal/modules/cjs/loader.js:650:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (C:\Users\****\AppData\Roaming\npm\node_modules\generator-upendodnn\generators\skinobject\index.js:4:14)
    at Module._compile (internal/modules/cjs/loader.js:702:30)    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
Emitted 'error' event at:
    at Immediate.setImmediate (C:\Users\***\AppData\Roaming\npm\node_modules\generator-upendodnn\node_modules\yeoman-generator\lib\index.js:433:22)
    at runCallback (timers.js:696:18)
    at tryOnImmediate (timers.js:667:5)    at processImmediate (timers.js:649:5)

**Comment by david-poindexter** [comment]
I am experiencing the same thing, but have not found a fix yet.

**Comment by david-poindexter** [comment]
It looks like this may be the culprit ("Error: Cannot find module 'uuid'):
https://github.com/UpendoVentures/generator-upendodnn/blob/eb576e5d4939d06ad872a20531805caa20bbc0ea/generators/skinobject/index.js#L4

**Comment by hismightiness** [comment]
The root cause is simply a typo in the skin object generator file.  It incorrectly requires `uuid` when it should be requiring `uuid-v4`.  

You can resolve this locally in your own environment as a quick fix by finding the following file.

`C:\Users\YOURUSERNAME\AppData\Roaming\npm\node_modules\generator-upendodnn\generators\skinobject\index.js`

Find this line of code:

```javascript
const uuid = require('uuid');
```

Replace that with this line of code:

```javascript
const uuid = require('uuid-v4');
```

**Comment by hismightiness** [comment]
This is fixed in release 01.02.01.  You can run the following command to get the latest updates.

`npm install -g generator-upendodnn`

**Comment by david-poindexter** [comment]
@hismightiness this is fantastic, I upgraded to latest (1.2.2) and will try it out soon.  Thank you!
